extensions/vscode/src/views/homeView.ts

```diff
@@ -268,7 +268,7 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
       deployProject(response.data.localId, this.stream);
     } catch (error: unknown) {
       const summary = getSummaryStringFromError("homeView, deploy", error);
-      window.showInformationMessage(`Failed to deploy . ${summary}`);
+      window.showErrorMessage(`Failed to deploy . ${summary}`);
```

Comment:
Not sure how this extra space snuck in, but lets fix it here.
```suggestion
      window.showErrorMessage(`Failed to deploy. ${summary}`);
```